app.py

```diff
@@ -0,0 +1,87 @@
+import base64
+from flask import Flask, request, render_template
+from flask_cors import cross_origin
+from image_utils import smile_detecting, face_as_net_input, extract_faces
+from keras.models import load_model
+import tensorflow as tf
+import argparse
+import numpy as np
+import cv2
+
+# creates a Flask application, named app
+app = Flask(__name__)
+
+
+cascpath = "haarcascade_frontalface_default.xml"
+modelpath = "model.hdf5"
+weightspath = "weights.03.hdf5"
+
+
+graph = tf.get_default_graph()
+model = load_model('model.hdf5')
+model.load_weights('weights.03.hdf5')
+print('loaded everything :)')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Input of model, weights and cascade path')
+    parser.add_argument('--cascade-path', '-c', required=False, type=str,
+                        help='OpenCV cascade file path, default set to: ' +
```

Comment:
No need to wrote what the default is. Argparser will write it. Additionally - why should we have a default param for that?

Reply:
done